### PR TITLE
Infer metadata automatically from package *.csv

### PR DESCRIPTION
**Pull request by Bachibouzouk**
Add code which is meant to be in oemof.tabular later to infer metadata automatically

### CODE REVIEW

File: examples/scenarios/wefe_pv_panel/scripts/infer.py [hunk 1]
```diff
@@ -3,36 +3,15 @@
...
 if "kwargs" not in locals():
     kwargs = {}
 
-# this automatically generates a datapackage.json file and the foreign keys should be
-# updated depending on the energy system design (works for existing facades, but for
-# new facades the additional 'add_foreign_keys_to_datapackage' function might be needed)
-building.infer_metadata(
+# this automatically generates a datapackage.json file
+otp_building.infer_metadata_from_data(
     package_name="wefe-pv-panel",
```
Comment (ciaradunks) [inline examples/scenarios/wefe_pv_panel/scripts/infer.py]: this is a big improvement!

File: src/oemof_tabular_plugins/datapackage/building.py [hunk 2]
```diff
@@ -1,37 +1,177 @@
...
+    r.commit()
+    return r
+
+
+def infer_package_foreign_keys(package):
+    """Infer the foreign_keys from data/elements and data/sequences and update meta data
+
+    Parameters
+    ----------
+    package
```
Comment (ciaradunks) [inline src/oemof_tabular_plugins/datapackage/building.py:110]: can you define what package is?

**Review by ciaradunks — APPROVED** [review]
This is really great that it can be automatically generated from the scenario input! Is there a way to automate it so that we do not need to run scripts/infer.py at all?

File: src/oemof_tabular_plugins/datapackage/building.py (hunk 2 above)
Comment (Bachibouzouk) [reply src/oemof_tabular_plugins/datapackage/building.py:110]: it is package from frictionless datapackage

**Comment by Bachibouzouk** [comment]
> This is really great that it can be automatically generated from the scenario input! Is there a way to automate it so that we do not need to run scripts/infer.py at all?

Yes one can just call this function in `compute.py`
[8045983](https://github.com/rl-institut/oemof-tabular-plugins/pull/17/commits/80459833d2594339e9263eb68eecd1988a90b7d9)

**Review by ciaradunks — APPROVED** [review]

**Pull request merged**